### PR TITLE
feat: implement `makePolicyContainer` and `clonePolicyContainer`

### PR DESCRIPTION
**Pull request by suneettipirneni**
Implements:
- https://html.spec.whatwg.org/multipage/origin.html#policy-container
- https://html.spec.whatwg.org/multipage/origin.html#clone-a-policy-container

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
What's the purpose of these? Isn't it better than we throw?

File: lib/fetch/util.js [hunk 1]
```diff
@@ -273,14 +273,36 @@ function createOpaqueTimingInfo (timingInfo) {
...
+  return {
+    CSPList: [],
+    embedderPolicy: {
+      value: 'unsafe-none',
+      reportingEndpoint: '',
+      reportOnlyValue: 'unsafe-none',
+      reportOnlyReportingEndpoint: ''
+    },
+    referrerPolicy: 'strict-origin-when-cross-origin'
+  }
```
Comment (ronag) [inline lib/fetch/util.js:285]: This is missing spec comments

**Comment by suneettipirneni** [comment]
> What's the purpose of these? Isn't it better than we throw?

Sorry, I'm unsure what you mean by this. Can you explain?

**Review by ronag — APPROVED** [review]

**Comment by mcollina** [comment]
What is the purpose of those functions in the context of undici? We do not have a CSP in Node.js, so why are they here?
IMHO we should not have those functions or throw in them when called.

**Pull request closed without merging**